packages/camera/camera_windows/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Update to latest camera platform interface, and fail if user attempts to use streaming with recording (since streaming is currently unsupported on web).
```

Comment:
Please use the present tense for release notes. Also `web` should be Windows here.

```suggestion
* Updates to latest camera platform interface but fails if user attempts to use streaming with recording (since streaming is currently unsupported on Windows).
```